tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/services/ServiceDeclTest.java

```diff
@@ -43,6 +45,13 @@ public void testServiceNameLiteral() {
         BRunUtil.invoke(compileResult, "testServiceName");
     }
 
+    @Test
+    public void testAttachMethodParams() {
+        CompileResult compileResult = BCompileUtil.compile("test-src/services/service_attach_test.bal");
+        BValue[] values = BRunUtil.invoke(compileResult, "testAttachMethodParams");
+        Assert.assertTrue(((BBoolean)values[0]).booleanValue());
```

Comment:
Shall we do this assertion in b7a itself?

Reply:
sure